### ISSUE TITLE
Values for engineer-* are inconsistent

### ISSUE THREAD

**Issue by ghost** [cmusphinx/cmudict#27]
I can't be sure of anything when it comes to phonetics as I'm no expert, but we've found it a big peculiar that some words starting with `engineer` have an inconsistent first value:

```
engineer EH1 N JH AH0 N IH1 R
engineer's EH2 N JH AH0 N IY1 R Z
engineered EH2 N JH AH0 N IY1 R D
engineering EH1 N JH AH0 N IH1 R IH0 NG
engineers EH1 N JH AH0 N IH1 R Z
engineers' EH1 N JH AH0 N IH1 R Z
```

Could it be that `EH2` is the more appropriate solution for all these first phonemes? Even with the secondary stress on the first phoneme, having multiple `1`-phonemes does mean we have to treat words has having more than one primary stress sometimes?

**Comment by danmartinez** [comment]
> having multiple 1-phonemes does mean we have to treat words has having more than one primary stress sometimes?
You're right, these are in error. Should only have one primary stress marked. As for EH1 vs EH2 in the first syllable, I'm inclined to agree that they should all be EH2. Perhaps there's dialectal distinction here I'm ignorant of.

**Comment by filipecatraia** [comment]
One thing I'm learning about linguistics, is that no one seems to agree on standardization 🤷 But as engineers, I'd propose we harmonize that `EH*` first phoneme, possibly always EH2.

**Comment by Alexir** [comment]
I would agree that two '1' marks seem odd but it does happen for some words, e.g. encyclopedia /ɪnˈsaɪkləˈpiːdiə/. 
It's mostly found in compound words; arguably 'engineering' is also a compound. 
Note that 'engineer' does have two pronunciation variants, American and RP. Also, N and V versions for some dialects. 

Probably it should be cleaned up, say separate things into "(1)" and "(2)" entries.
